nearcore/src/runtime/mod.rs

```diff
@@ -1377,9 +1375,8 @@ mod test {
 
             let shard_uid =
                 self.epoch_manager.shard_id_to_uid(shard_id, &EpochId::default()).unwrap();
-            if let Some(flat_storage) = self
```

Comment:
note: no meaningful change below in this file other than stop checking return value of get_flat_storage_manager()